sandBox/FirstSprint/TaskLoops.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Text;
+
+namespace FirstSprint
+{
+    class TaskLoops
+    {
+        private string[] GetRandomStringArray()
+        {
+            int counter = 0;
+            string alphabet = "QWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnm";
+            List<string> array = new List<string>();
+            Random random = new Random();
+
+
+            while (counter <= random.Next(5,100))
+            {
+                StringBuilder sb = new StringBuilder(random.Next(2, 30) - 1);
+                int position = 0;
+
+                for (int i = 0; i < random.Next(2, 30); i++)
+                {
+                    position = random.Next(0, alphabet.Length - 1);
+                    sb.Append(alphabet[position]);
+                }
+
+                array.Add(sb.ToString());
+                counter++;
+            }
+
+            return array.ToArray<string>();
+
+        }
+
+        public Dictionary<int, string> GetNewDictionary(string[] inputStringArray)
```

Comment:
Can be replaced by something from here: https://stackoverflow.com/questions/15252225/convert-an-array-to-dictionary-with-value-as-index-of-the-item-and-key-as-the-it